### ISSUE TITLE
Remove blank behaviour in different cases

### ISSUE THREAD

**Issue by velios** [oakmac/standard-clojure-style-js#156]
Hi, just watched your conj talk, this is cool - your efforts are appreciated.

Came across two different cases in different editors
Pic instead code block because it shows blankspaces.

In first case, which by default in my vscode setup
<img width="108" alt="image" src="https://github.com/user-attachments/assets/df5c8ebe-7c8c-4653-b8d2-efe7cd52514f">
will be formatted to 
```clojure
(cond
  :cond-1
  :do-1
  :cond-2
  :do-2
  :cond-3
  :do-)
```

but(which default in my cursive setup)
<img width="119" alt="image" src="https://github.com/user-attachments/assets/d481d2c7-382b-464c-81fa-61f7e65d12db">
will be formatted with respect to blank lines to
```clojure
(cond
  :cond-1
  :do-1

  :cond-2
  :do-2

  :cond-3
  :do-)
```

I would personally prefer to respect empty lines, but I wonder which behavior is considered preferable and whether the formatter behaves correctly in both cases?

**Comment by oakmac** [comment]
> Hi, just watched your conj talk, this is cool - your efforts are appreciated.

Thank you! :grin: 

<hr>

I tested this as input and the formatter returned it unchanged (which is correct behavior):

```clj
(cond
  some-condition
  12

  another-condition
  13

  :else
  nil)
```

Can share the file exactly? What version of Standard Clojure Style are you using?

Another hunch: are you on Windows and does this file use CRLF line endings? The formatter should be able to handle that, but there was another bug that looked similar to this and I wonder if this is somehow related ([Issue #102](https://github.com/oakmac/standard-clojure-style-js/issues/102) and [Issue #103](https://github.com/oakmac/standard-clojure-style-js/issues/103)).

**Comment by velios** [comment]
Of course, my fault. Example https://github.com/velios/standart-clojure-style-issue
I'm on mac, compile clojurefmt to standalone binary like you show on video

**Comment by oakmac** [comment]
Thank you sharing that!

Adding this here for my debugging efforts. I do see a difference between `with_spaces.clj` and `without_spaces.clj` when viewing with a hex editor.

<img width="1461" alt="Screenshot 2024-11-11 at 9 38 49 AM" src="https://github.com/user-attachments/assets/94f37eca-f510-4f80-b917-e7009be4961a">

**Comment by john-shaffer** [comment]
I have noticed this on linux as well. "Blank" lines with spaces on them are removed, but actually empty lines with no characters are retained.
